recipes-core/busybox/busybox_%.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+SRC_URI:append = "\
+    file://tdx-enc-handler-requirements.cfg \
```

Comment:
This config fragment is only required if encryption is enabled, right?

If so, you can do something like this:

```
SRC_URI:append = "\
    ${@oe.utils.conditional('TDX_ENC_ENABLE', '1', 'file://tdx-enc-handler-requirements.cfg', '', d)} \
"
```